nixos/lib/make-options-doc/mergeJSON.py

```diff
@@ -208,19 +236,37 @@ def optionIs(option: Dict[str, Any], key: str, typ: str) -> bool:
         if '_type' not in option[key]: return False
         return option[key]['_type'] == typ
 
+    def convertCode(name: str, option: Dict[str, Any], key: str):
+        if optionIs(option, key, 'literalMD'):
+            option[key] = convertString(name, f"*{key.capitalize()}:*\n{option[key]['text']}")
+        elif optionIs(option, key, 'literalExpression'):
+            code = option[key]['text']
+            if '\n' in code:
+                docbook = convertString(name, f"```\n{code}\n```")
+                option[key] = f"<para><emphasis>{key.capitalize()}:</emphasis></para> {docbook}"
+            else:
+                option[key] = convertString(name, f"*{key.capitalize()}:* ` {code} `")
```

Comment:
```suggestion
                option[key] = convertString(name, f"*{key.capitalize()}:* `{code}`")
```

We might also want to check that `code` doesn't contain backticks